alongtheway.txt

```diff
@@ -501,11 +501,11 @@ Read these parables and write down how you feel and what stands out as you read
 
 WEEK SEVEN: MARY
 
-As I'm writing this we're just coming to the end of the Christmas season and my favorite songs of this season are the songs about Mary and Joseph - exploring the humanity within the Nativity story. Over the last month or two I've thought a lot about Mary and her journey. I wanted to explore her story together and I have been praying that you would come to this story with a fresh perspective and leave on the other side refreshed and strengthened for whatever God has been calling you to on your own journey with him.
+As I'm writing this we're just coming to the end of the Christmas season and my favorite songs of this season are the songs about Mary and Joseph - exploring the humanity within the Nativity story. Over the last month or two I've thought a lot about Mary and her journey. I wanted to explore her story together and I have been praying that you would come to this story with a new perspective and leave on the other side refreshed and strengthened for whatever God has been calling you to on your own journey with him.
```

Comment:
Sure!

Sent from my iPhone

> On Jan 15, 2016, at 11:13 PM, Joshua Maxwell <notifications@github.com> wrote:
> 
> In alongtheway.txt:
> 
> > @@ -501,11 +501,11 @@ Read these parables and write down how you feel and what stands out as you read
> >  
> >  WEEK SEVEN: MARY
> >  
> > -As I'm writing this we're just coming to the end of the Christmas season and my favorite songs of this season are the songs about Mary and Joseph - exploring the humanity within the Nativity story. Over the last month or two I've thought a lot about Mary and her journey. I wanted to explore her story together and I have been praying that you would come to this story with a fresh perspective and leave on the other side refreshed and strengthened for whatever God has been calling you to on your own journey with him.
> > +As I'm writing this we're just coming to the end of the Christmas season and my favorite songs of this season are the songs about Mary and Joseph - exploring the humanity within the Nativity story. Over the last month or two I've thought a lot about Mary and her journey. I wanted to explore her story together and I have been praying that you would come to this story with a new perspective and leave on the other side refreshed and strengthened for whatever God has been calling you to on your own journey with him.
> replace "fresh perspective" with "new perspective" since we use "refreshed" later...maybe overkill on my part? what do you think?
> 
> —
> Reply to this email directly or view it on GitHub.
>